### PR TITLE
unmount all mounts for a particular drive and doc updates

### PR DESCRIPTION
**Pull request by wlan0**

### CODE REVIEW

File: docs/architecture.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+The 4 components run as two different pods. 
+
+| Name                          | Components                                           | Description                        |
+|-------------------------------|------------------------------------------------------|------------------------------------|
+| DirectCSI Node Driver         | CSI Driver, Driver Controller, Volume Controller     | runs on every node as a DaemonSet  |
+| DirectCSI Central Controller  | CSI Controller                                       | runs as a deployment               |
+
+
+### Scalability
```
Comment (Praveenrajmani) [inline docs/architecture.md:27]: Can we say that the default replica count for `DirectCSI Central Controller` is 3?

File: docs/index.md [hunk 2]
```diff
@@ -9,13 +9,16 @@ Documentation
  - [Installation](./installation.md)
  - [CLI reference](./cli.md)
 
+### Advanced
+ - [Architecture](./architecture.md)
+ - [Security](./security.md)
+ - [CSI Specification](./specification.md)
+
```
Comment (Praveenrajmani) [inline docs/index.md:16]: Can we also mention the minimum version requirements? Like, Kubernetes version, sidecar container versions [csi-node-driver-registrar, csi-provisioner, csi-driver, etc]

Comment (wlan0) [reply docs/index.md:16]: Good point

Comment (wlan0) [reply docs/index.md:16]: I've added it to installation docs

**Pull request merged**